src/health_manager.py

```diff
@@ -48,7 +48,7 @@ def reset_chicken_flag(self):
 
     def update_location(self, loc: Location):
         if loc is not None and type(loc) == str:
-            bosses = ["shenk", "eldritch", "pindle", "nihlatak", "trav"]
+            bosses = ["shenk", "eldritch", "pindle", "nihlatak", "trav", "arcane"]
```

Comment:
Looks like this should be "arc" instead to match locations such as "a2_arc_start". I was wondering why the potions are not working :)

Reply:
thanks